.pre-commit-config.yaml

```diff
@@ -1,7 +1,34 @@
+default_stages: [commit]
+
 repos:
-  - repo: https://github.com/gitguardian/ggshield
```

Comment:
@aeswibon we definitely don't want to remove `ggshield`

Reply:
Okay, I will revert that.